Makefile

```diff
@@ -76,3 +85,7 @@ create_hsm_key: .script-build-dirs .run_create_hsm_key .script-clean-dirs
 # Command: Derive xPub from seed
 .PHONY: xpub_from_seed
 xpub_from_seed: .script-build-dirs .run_xpub_from_seed .script-clean-dirs
+
+# Command: Derive multisig xPub from seed
+.PHONY: multisig_xpub_from_seed
```

Comment:
question(non-blocking): Should we add information about this to our `readme.md` file?

Reply:
UPDATE: Actually, I found a suitable location for it: https://github.com/HathorNetwork/hathor-wallet-headless/pull/520/commits/cf410e20d6008ef0dd61d98ff8c3d85aac303df6